src/main/java/ch/naviqore/raptor/router/Query.java

```diff
@@ -9,10 +9,10 @@
 import java.util.*;
 
 /**
- * The objective stores the progress of the raptor algorithm. Each request needs a new objective instance.
+ * The query stores the progress of the raptor algorithm. Each request needs a new query instance.
```

Comment:
The query stores the configuration of the raptor routing request and holds working variables to store the progress of the raptor algorithm.